src/api/app/jobs/send_event_emails.rb

```diff
@@ -1,10 +1,18 @@
-class SendEventEmails < CreateJob
-
-  attr_accessor :event
+class SendEventEmails
 
   def perform
-    subscribers = event.subscribers
-    return if subscribers.empty?
-    EventMailer.event(subscribers, event).deliver_later
+    Event::Base.where(mail_sent: false).order(:created_at).limit(100).each do |event|
+      self.mail_sent = true
```

Comment:
actuallly the migration creates mail*s*_sent - good thing I added a test case ;)